src/config/index.ts

```diff
@@ -87,6 +87,8 @@ export const getIntercomId = () =>
 
 export const getExchangeRatesUrl = () => 'https://api.exchangeratesapi.io/latest'
 
+export const getExchangeRatesUrlFallback = () => 'https://api.coinbase.com/v2/exchange-rates'
```

Comment:
Why did you add this fallback?

Reply:
Because `exchangeratesapi ` does not provide the ETH price, coinbase yes, instead of replacing the first with the second I thought it was a good idea to have a fallback in case the first one does not have the values or it's down

Reply:
Where do we get the ETH price currently, then? 
![image](https://user-images.githubusercontent.com/17196842/87804385-172f3800-c854-11ea-9e94-06c7c8e5780c.png)

Reply:
From the backend, now that I see it I could just use that, won't be direct as is it now but it could work, should I remove the fallback? what do you think?

Reply:
Yes, if there is no strong reason to add it, I would suggest to remove the fallback. It just adds a dependency on Coinbase, even if it's just a fallback.

Reply:
Was reviewing this, we would need the endpoint because we are getting the balance of ether in USD from the backend using this endpoint: `https://safe-transaction.staging.gnosisdev.com/api/v1/safes/safeAddress/balances/usd/`

And that could be used to get the value in USD of the Ether, but if the user does not have ether in the safe, it will return 0. If the user has for example 100 GNO and 0 ETH he only be able to see the balances in USD/CAD/EUR/ETC but not in ether

Reply:
Okay. Maybe @Uxio0 has some thoughts whether it would make sense to also somehow return the ETH price in our backend. But in that case I am fine going with the above endpoint for now, as our backend current indeed does not allow to cover this case of a user having 0 ETH but some ERC-20 tokens and the value denominated in ETH.

Reply:
As I think finding a user with 0Eth is not very common we shouldn't change anything in the backend for now

Reply:
hey @Uxio0 I have a safe w/o Ether, just tokens for now. are we sure this is a 'rare' case?

Reply:
This will become more common when the ui starts using the relay service, and then we can think about that, but for now I would say that's not common. Either way I don't see the transaction service the place to query about ether price

Reply:
What do you suggest to do @lukasschor ?

Reply:
Even without the relay service, there is no need for users to have ETH on their Safe, as gas fees are paid with the signer wallet. So I would not consider this as an edge case. Nevertheless, I agree with @Uxio0 that we should not really rely on the tx service for the ETH price, so I wouldn't be adding a price oracle in the backend. Given these circumstances I think it's fine getting the ETH/USD conversion from coinbase. We can think about implementing a more decentralized solution (such as Chainlink) later.

Reply:
If we would allow to create transactions using a fiat value (E.g. "Send $5 in ETH to recipient X") This would be more critical. But as we just use fiat conversion to give a better picture of the value of one's assets, Coinbase should do the job for now.

Reply:
Ok makes sense, will do that, later on we could just create a ticket to improve the oracle solution, thanks!